content/en/docs/Examples/boot_assessment_trusted_boot.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Enabling Automatic Boot Assessment with Trusted Boot"
+linkTitle: "Enabling Automatic Boot Assessment with Trusted Boot"
+weight: 4
+description: This section describe examples on how to enable Automatic Boot Assessment with Trusted Boot in your own services.
+---
+
+
+# Enabling Automatic Boot Assessment with Trusted Boot
+
+In this tutorial, we will walk through how to configure Kairos to enable **automatic boot assessment**, where the boot loader can determine the health of a boot entry. Specifically, we'll configure systemd services to trigger the `boot-complete.target` to mark boot entries as *good* or *bad*. We'll also cover how to implement automatic reboots when a service fails, allowing retries of a boot entry until success or exhaustion of attempts.
+
+---
+
+
+## Overview of Trusted Boot Automatic Boot Assessment
+
+`systemd-boot` manages boot entries and provides a mechanism to automatically assess their success or failure. The key features are:
+
+1. **Boot Entry Marking**:
+    - A boot entry is marked as *good* when the `boot-complete.target` is reached during startup.
+    - If the system fails to reach `boot-complete.target`, the boot entry is marked as *bad*.
+
+2. **Retries**:
+    - By default, each boot entry has **3 retries**. A failure to reach `boot-complete.target` reduces the retry count. Once retries are exhausted, another boot entry is chosen if available.
+
+3. **Service-Level Controls**:
+    - Configure services to participate in boot entry assessment and trigger reboots on failure.
+
+---
+
+## Step 1: Configuring a Service to Trigger `boot-complete.target`
+
+To ensure a service's success or failure impacts the boot assessment, modify its service file to interact with `boot-complete.target`:
+
+1. **Edit the Service File**:  
+
+   Override the service configuration using:
+   ```bash
+   sudo systemctl edit <service-name>
```

Comment:
mmh where this commands has to be run? in a running system? how can we do that from the cloud config file?

Reply:
incoming in a sec!